2025-01-28 18:24:34,980 INFO    Thread-3  :305903 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_setup.py:_flush():76] Configure stats pid to 305903
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_setup.py:_flush():76] Loading settings from /home/ale/.config/wandb/settings
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_setup.py:_flush():76] Loading settings from /home/ale/project-sim2real-montana-scavone/Extension_walker_2D/UDR/wandb/settings
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'custom-walker-ppo', 'entity': 'alex-scavone-politecnico-di-torino', 'root_dir': '/home/ale/project-sim2real-montana-scavone/Extension_walker_2D/UDR', 'sweep_id': 'ay1m2gs9', 'run_id': 'xeci5hb4', 'sweep_param_path': '/home/ale/project-sim2real-montana-scavone/Extension_walker_2D/UDR/wandb/sweep-ay1m2gs9/config-xeci5hb4.yaml'}
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Extension_walker_2D/UDR/train.py', 'program_abspath': '/home/ale/project-sim2real-montana-scavone/Extension_walker_2D/UDR/train.py', 'program': '/home/ale/project-sim2real-montana-scavone/Extension_walker_2D/UDR/train.py'}
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_init.py:_log_setup():521] Logging user logs to ./logs_train/run_0/wandb/run-20250128_182434-xeci5hb4/logs/debug.log
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_init.py:_log_setup():522] Logging internal logs to ./logs_train/run_0/wandb/run-20250128_182434-xeci5hb4/logs/debug-internal.log
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_init.py:init():561] calling init triggers
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_init.py:init():568] wandb.init called with sweep_config: {'delta_foot_mass': '0.25', 'delta_leg_mass': '0.35', 'delta_thigh_mass': '0.2'}
config: {}
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_init.py:init():611] starting backend
2025-01-28 18:24:34,981 INFO    Thread-3  :305903 [wandb_init.py:init():615] setting up manager
2025-01-28 18:24:34,984 INFO    Thread-3  :305903 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-01-28 18:24:34,986 INFO    Thread-3  :305903 [wandb_init.py:init():623] backend started and connected
2025-01-28 18:24:34,989 INFO    Thread-3  :305903 [wandb_run.py:_config_callback():1347] config_cb None None {'delta_foot_mass': '0.25', 'delta_leg_mass': '0.35', 'delta_thigh_mass': '0.2'}
2025-01-28 18:24:34,990 INFO    Thread-3  :305903 [wandb_init.py:init():715] updated telemetry
2025-01-28 18:24:34,996 INFO    Thread-3  :305903 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2025-01-28 18:24:35,510 INFO    Thread-3  :305903 [wandb_run.py:_on_init():2357] communicating current version
2025-01-28 18:24:35,572 INFO    Thread-3  :305903 [wandb_run.py:_on_init():2366] got version response upgrade_message: "wandb version 0.19.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-01-28 18:24:35,573 INFO    Thread-3  :305903 [wandb_init.py:init():799] starting run threads in backend
2025-01-28 18:24:36,110 INFO    Thread-3  :305903 [wandb_run.py:_console_start():2335] atexit reg
2025-01-28 18:24:36,110 INFO    Thread-3  :305903 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2025-01-28 18:24:36,111 INFO    Thread-3  :305903 [wandb_run.py:_redirect():2255] Wrapping output streams.
2025-01-28 18:24:36,111 INFO    Thread-3  :305903 [wandb_run.py:_redirect():2280] Redirects installed.
2025-01-28 18:24:36,111 INFO    Thread-3  :305903 [wandb_init.py:init():842] run started, returning control to user process
2025-01-28 18:24:36,112 INFO    Thread-3  :305903 [wandb_run.py:_finish():2064] finishing run alex-scavone-politecnico-di-torino/custom-walker-ppo/xeci5hb4
2025-01-28 18:24:36,112 INFO    Thread-3  :305903 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 1
2025-01-28 18:24:36,112 INFO    Thread-3  :305903 [wandb_run.py:_restore():2287] restore
2025-01-28 18:24:36,112 INFO    Thread-3  :305903 [wandb_run.py:_restore():2293] restore done
2025-01-28 18:24:39,885 INFO    Thread-3  :305903 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2025-01-28 18:24:39,885 INFO    Thread-3  :305903 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2025-01-28 18:24:39,889 INFO    Thread-3  :305903 [wandb_run.py:_footer_sync_info():3895] logging synced files
